lib/vero/utility/logger.rb

```diff
@@ -25,6 +25,10 @@ def logger
             nil
           end
         end
+
+        def logging_enabled?
+          logger && (Vero::App.default_context.config.logging || defined(RSpec))
```

Comment:
does that `defined` need to be `defined?`

Reply:
ah yes you're right, good spotting 👍